### PR TITLE
Update NDK version to r26d in docs

### PR DESCRIPTION
**Pull request by kirklandsign**

### CODE REVIEW

File: examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ Phone verified: OnePlus 12. Samsung S23 (Llama only), Samsung S24+ (Llama only),
...
+* Install the [Android SDK API Level 34](https://developer.android.com/about/versions/15/setup-sdk) and [Android NDK r26d](https://developer.android.com/studio/projects/install-ndk).
 * If you have Android Studio set up, you can install them with
   * Android Studio Settings -> Language & Frameworks -> Android SDK -> SDK Platforms -> Check the row with API Level 34.
   * Android Studio Settings -> Language & Frameworks -> Android SDK -> SDK Tools -> Check NDK (Side by side) row.
 * Alternatively, you can follow [this guide](https://github.com/pytorch/executorch/blob/856e085b9344c8b0bf220a97976140a5b76356aa/examples/demo-apps/android/LlamaDemo/SDK.md) to set up Java/SDK/NDK with CLI.
 Supported Host OS: CentOS, macOS Sonoma on Apple Silicon.
 
 
-Note: This demo app and tutorial has only been validated with arm64-v8a [ABI](https://developer.android.com/ndk/guides/abis), with NDK 25.0.8775105.
+Note: This demo app and tutorial has only been validated with arm64-v8a [ABI](https://developer.android.com/ndk/guides/abis), with NDK r26d.
```
Comment (cmodi-meta) [inline examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md:25]: @kirklandsign , can we specifically call out the version number (v26.3.11579264)?

Comment (kirklandsign) [reply examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md:25]: I feel like r26d is the official name from android pages. The version number v26.3.11579264 only appears on sdkmanager

Comment (kirklandsign) [reply examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md:25]: But let me know if you think v26.3.11579264 is more accurate

Comment (cmodi-meta) [reply examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md]: I made a commit on top of yours to address this.

Comment (cmodi-meta) [reply examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md]: oops sorry, my page didn't refresh so just saw this now. Just to keep format of previous the same, are you ok with v26.3.11579264 then we continue with this?

Comment (kirklandsign) [reply examples/demo-apps/android/LlamaDemo/docs/delegates/xnnpack_README.md]: Yes! v26.3.11579264 works for me!

**Review by cmodi-meta — APPROVED** [review]

**Review by cccclai — APPROVED** [review]

**Pull request closed without merging**